### ISSUE TITLE
Vulkan shader pre-caching takes much more time with new NVIDIA driver.

### ISSUE THREAD

**Issue by deathxxx123** [ValveSoftware/steam-for-linux#7571]
#### Your system information

* Steam client version (build number or date): i don't know this, it's lastest from Ubuntu rep
* Distribution: Ubuntu Budgie 20.04
* Opted into Steam client beta?: Yes and No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large code pastes as a [Github Gist](https://gist.github.com/)
With nvidia 450.80.02, all are ok.
With nvidia 455.45.01, shader pre-caching takes much more time, like 10x or more.

#### Steps for reproducing this issue:

1. Install nvidia-driver-455 from ubuntu drivers PPA
2. Start some big game with Proton
3. See how long will take to preload vulkan shaders.

I'm forced to stay with old LTS nvidia driver to get fast speed shader preloading.
That not only mater before start the games, but also when playing any game with Proton.
Is there anyone who have no problem with the new Nvidia driver?

**Comment by gbschenkel** [comment]
Try use the beta driver 460.27.04.
https://www.nvidia.com/download/driverResults.aspx/167671/en-us

I am using without flaws.

**Comment by deathxxx123** [comment]
> Try use the beta driver 460.27.04.
> https://www.nvidia.com/download/driverResults.aspx/167671/en-us
> 
> I am using without flaws.

How you find this in Nvidia site? It is not listed in normal search....

**Comment by gbschenkel** [comment]
Yes, the first time I went into the site I didn't find too. The site have some problems, expecially if you click straight in the [GForce Drivers](https://www.nvidia.com/en-us/geforce/drivers/)
![image](https://user-images.githubusercontent.com/1715069/103350988-84402e00-4a80-11eb-8cb0-dce9a7927862.png)
You need click on [All NVidia Drivers](https://www.nvidia.com/Download/index.aspx) instead.

On bottom you will find the "Beta and Older Drivers"
![image](https://user-images.githubusercontent.com/1715069/103351163-121c1900-4a81-11eb-9f5c-4b0215541830.png)

**Comment by damian101** [comment]
Vulkan shader processing time has increased to an absurd amount for me now, yesterday I wanted to play Counter Strike, and after 6 hours I just skipped it. Medium-range Alder Lake laptop CPU. Doesn't take that long for other games, though.
